### PR TITLE
cleanup local env setup and rename api env variables

### PR DESCRIPTION
**Pull request by karlandindrakryggen**
* cleanup local env setup 
* rename env variables for KursinfoAPI to include Kursinfo in the name (eg API_URI -> KURSINFO_API_URI)

https://kth-se.atlassian.net/browse/KUI-1316

### CODE REVIEW

File: .env.in [hunk 1]
```diff
@@ -1,19 +1,6 @@
...
-OIDC_CALLBACK_URL=<not needed if localhost>
-OIDC_CALLBACK_SILENT_URL=<not needed if localhost>
-OIDC_CALLBACK_LOGOUT_URL=<not needed if localhost>
-REDIS_URI=[connection string to redis]
+KURSINFO_API_KEY=[Available in Azure KeyVault]
+OIDC_APPLICATION_ID=[Available in Azure KeyVault]
+OIDC_CLIENT_SECRET=[Available in Azure KeyVault]
+REDIS_URI=[Available in Azure KeyVault]
+
 BLOB_SERVICE_SAS_URL=https://[common storage name].blob.core.windows.net/?[params]&spr=https&sig=[generated signature]
```
Comment (karlandindrakryggen) [inline .env.in:6]: Osäker på denna (dvs BLOB_SERVICE_SAS_URL) borde kanske kunna ha en värde i `devDefaults()` också?

File: .azure/prod.parameters.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "secretNames": {
       "value": [
         "ANALYSES_STORAGE_URL",
-        "API_KEY",
+        "KURSINFO_API_KEY",
```
Comment (karlandindrakryggen) [inline .azure/prod.parameters.json:11]: Denna behöver uppdateras i KeyVault också innan deploy

File: .env.in (hunk 1 above)
Comment (belanglos) [reply .env.in:6]: Not sure I understand, I thinkg BLOB_SERVICE_SAS_URL should be in the KeyVault as it is a signed URL with access rights, right?

File: README.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ localhost:3000/kursinfoadmin/kurser/kurs/:courseCode
 ```
 
 - Course introduction text for course information page, which can be edited by course coordinators and examiners.
-  (requires only: `API_KEY` kursinfo-api)
+  (requires only: `KURSINFO_API_KEY` kursinfo-api)
```
Comment (belanglos) [inline README.md:29]:
```suggestion
```

File: .env.in (hunk 1 above)
Comment (belanglos) [inline .env.in:6]:
```suggestion
BLOB_SERVICE_SAS_URL=[Available in Azure KeyVault]
```

**Review by belanglos — APPROVED** [review]

**Review by belanglos — APPROVED** [review]

File: .azure/prod.parameters.json (hunk 2 above)
Comment (karlandindrakryggen) [reply .azure/prod.parameters.json:11]: Inlagt i KeyVault nu (både ref och prod)

**Pull request merged**